clients/client-acm-pca/waiters/waitForAuditReportCreated.ts

```diff
@@ -42,3 +47,16 @@ export const waitForAuditReportCreated = async (
   const serviceDefaults = { minDelay: 3, maxDelay: 120 };
   return createWaiter({ ...serviceDefaults, ...params }, input, checkState);
 };
+/**
+ * Wait until a Audit Report is created
+ *  @param params : Waiter configuration options.
+ *  @param input : the input to DescribeCertificateAuthorityAuditReportCommand for polling.
+ */
```

Comment:
`@param` is not standard, not sure if doc generator can recognize it. Here's [the annotation example](https://tsdoc.org/pages/tags/param/)
```suggestion
/**
 * Wait until a Audit Report is created
 *  @param params - Waiter configuration options.
 *  @param input - the input to DescribeCertificateAuthorityAuditReportCommand for polling.
 */
```